zone/bonuses.cpp

```diff
@@ -75,6 +76,7 @@ void NPC::CalcBonuses()
 
 void Client::CalcBonuses()
 {
+	Message(Chat::Yellow, "Calculating bonuses.");
```

Comment:
This should be removed

Reply:
Suggest LogSpells or creating a new category that spits out bonus calculations, might make your guys' life easier debugging bonuses later down the road cc @Aeadoin

Reply:
Woops. Left a debug message in. Thought I got them all.